### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by sagar455**
landing-page

ps: not responsive yet

image

documentation

image

### CODE REVIEW

**Comment by Husain01** [comment]
Could you add a CSB link for easy reviewing.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        </div>
+    </header>
+    
+    <div class="container">
+        <div class="content">
+            <img class="content-logo" src="logo.svg" alt="">
+            <p>The Classy UI That You Need.</p>
+            <div class="content-links">
+            <button class="btn btn-grey"><a href="./Documentation/installation-doc/installation.html">Get Started</a></button>
+            <button class="btn btn-grey"><a href="#">GitHub</a></button>
```
Comment (Husain01) [inline index.html:30]: Buttons are not clickable because the image overflows over it.

**Review by Husain01 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (AtulPant2704) [reply index.html:30]: Yes either decrease the size of image or give it z-index=-1, maybe it would work then.

Comment (sagar455) [reply index.html:30]: its fixed. i have given top margin

File: Documentation/doc/doc.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    background-color: var(--grey-300);
+    padding-top: 20px;
+  }
+  
+  .sidenav h3{
+    padding-top:10px;
+  }
+  
+  .sidenav a {
+    padding: 6px 8px 20px 16px;
```
Comment (Sudhanshusoni45) [inline Documentation/doc/doc.css]: Suggestion : Use rems instead of px will help in responsiveness

Comment (sagar455) [reply Documentation/doc/doc.css]: okki will change it

**Comment by Sudhanshusoni45** [comment]
Suggestion : You can give some breathing space ( padding left-right) in the sidebar

**Review by Sudhanshusoni45 — APPROVED** [review]
Suggestion : your background color and button background color, has low contrast ratio you can increase that for better UX

**Pull request merged**